crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -495,7 +468,6 @@ TEST(URandomTest, Test) {
     TRACE_FLAG(URANDOM_ERROR);
 
     const std::vector<Event> expected_trace = TestFunctionPRNGModel(flags);
```

Comment:
Are there any calls we can capture from Jitter entropy that we can expand our PRNG model with? Right now we are not actually modeling any Jitter calls.

Reply:
response in the previous comment